crates/apollo-compiler/CHANGELOG.md

```diff
@@ -17,28 +17,14 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 ## Maintenance
 ## Documentation-->
 
-<!--
 # [x.x.x] (unreleased) - 2023-mm-dd
 
-## Features
-- Add `validate_standalone_executable` function to validate an executable document without access to a schema, by [goto-bus-stop] in [pull/631], [issue/629]
```

Comment:
This was commented out for some reason, but it’s a real feature that was added in 0.11.2